### PR TITLE
deprecated

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: docker/pykafka/build.conf [hunk 1]
```diff
@@ -1 +1,3 @@
 version=1.0.0
+deprecated=true
+deprecated_reason=Use the confluent-kafka image instead
```
Comment (JudahSchwartz) [inline docker/pykafka/build.conf:3]:
```suggestion
deprecated_reason=Use the confluent-kafka image instead.
```

File: docker/pymisp/build.conf [hunk 2]
```diff
@@ -1 +1,3 @@
 version=1.0.0
+deprecated=true
+deprecated_reason=Use the pymisp2 image instead
```
Comment (JudahSchwartz) [inline docker/pymisp/build.conf:3]:
```suggestion
deprecated_reason=Use the pymisp2 image instead.
```

File: docker/threatconnect-sdk/build.conf [hunk 3]
```diff
@@ -1 +1,3 @@
 version=1.0.0
+deprecated=true
+deprecated_reason=Use the threatconnect-py3-sdk instead
```
Comment (JudahSchwartz) [inline docker/threatconnect-sdk/build.conf:3]:
```suggestion
deprecated_reason=Use the threatconnect-py3-sdk instead.
```

**Review by tkatzir — APPROVED** [review]
Great!

File: docker/dempcap/build.conf [hunk 4]
```diff
@@ -1,2 +1,4 @@
 version=1.0.0
 ironbank=true
+deprecated=true
+deprecated_reason=All scripts using this image were deprecated.
```
Comment (JudahSchwartz) [inline docker/dempcap/build.conf:4]:
```suggestion
```

File: docker/dempcap/build.conf [hunk 5]
```diff
@@ -1,2 +1,2 @@
 version=1.0.0
-ironbank=true
+ironbank=true
```
Comment (JudahSchwartz) [inline docker/dempcap/build.conf:2]:
```suggestion
ironbank=true```

**Pull request merged**